encrypted-media-respec.html

```diff
@@ -3172,7 +3213,7 @@
 
         <section id="allow-identifiers-cleared">
           <h3>Allow Identifiers to Be Cleared</h3>
-          <p>As a consequence of the requirements in <a href="#allow-persistent-data-cleared">Allow Persistent Data to Be Cleared</a>, implementations that use <a def-id="distinctive-identifier">Distinctive Identifier(s)</a> MUST allow the user to clear those identifiers such that they are no longer retrievable both outside, such as via the APIs defined in this specification, and on the client device.
+          <p>As a consequence of the requirements in <a href="#allow-persistent-data-cleared">Allow Persistent Data to Be Cleared</a>, implementations that <a href="#uses-distinctive-identifiers-or-distinctive-permanent-identifiers">Distinctive Identifier(s) or Distinctive Permanent Identifier(s)</a> MUST allow the user to clear the <a def-id="distinctive-identifier">Distinctive Identifier(s)</a> and values associated with <a def-id="distinctive-permanent-identifier-maybe-plural"></a> such that they are no longer retrievable both outside, such as via the APIs defined in this specification, and on the client device.
```

Comment:
Distinctive Permanent Identifiers must not be Distinctive Identifiers because the latter are clearable, per-origin, and may be exposed to the application. Thus, we need to ensure the definitions don't imply otherwise.

It may also be subtle, but I don't believe the current definitions mean Distinctive Permanent Identifiers are Distinctive Identifiers (even though they are "distinctive" and "identifiers"). The first two sub-bullets in the Distinctive Identifier definition require derivation or generation. Permanent Identifiers are not generated or derived, at least within the scope of this specification. (They are generated in a factory and may be generated or derived as part of, for example, an OS installation, but I don't think that's what we are referring to.)

For now, I have added a NOTE to clarify this, but it relies on that same "scope" argument. Perhaps we can improve the wording to avoid any question.

One option might be to add "Is not a [Distinctive] Permanent Identifier" to the list of ANDed properties of Distinctive Identifier.